src/SoundManager.h

```diff
@@ -5,34 +5,119 @@
 #include <SFML/Audio/Sound.hpp>
 #include <SFML/Audio/Music.hpp>
 
+#include <unordered_map>
+
 class SoundManager
 {
 private:
+    std::unordered_map<std::string, sf::SoundBuffer> soundBuffers;
+    std::unordered_map<uint8_t, sf::Sound*> sounds;
```

Comment:
O acesso é mais rápido mas gasta um pouco mais de memória. Vi um outro exemplo de Sound Manager que fez desse jeito. Apenas segui mesmo haha

**map**

Usually implemented using red-black tree.
Elements are sorted.
Relatively small memory usage (doesn't need additional memory for the hash-table).
Relatively fast lookup: O(log N).

**unordered_map**

Usually implemented using hash-table.
Elements are not sorted.
Requires additional memory to keep the hash-table.
Fast lookup O(1), but constant time depends on the hash-function which could be relatively slow. Also keep in mind that you could meet with the Birthday problem.